lib/middleware.js

```diff
@@ -43,6 +43,16 @@ export function createGetKeystoreConfig({routes}) {
   });
 }
 
+// creates middleware to get all the keystore configs for root controller
+export function createGetAllKeystoreConfigs() {
+  return asyncHandler(async (req, _, next) => {
+    const controller = req.query.controller;
+    const results = await storage.getAll({req, controller});
+    req.keystores = results.map(r => r.config);
+    next();
+  });
+}
```

Comment:
This seems like an odd middleware. I don't think we need to fetch all configs in a middleware. The middleware that fetches a single config in a middleware exists to enable authz checks and run KMS ops, but that's not a pattern to copy otherwise.

So I think we ought to just call this code from the main handler. Additionally, the main handler is eventually going to need to support other query params for pagination, so the API is a little off.

I think we want the `getAll()` API above to look more like it does in br-service-core:

https://github.com/digitalbazaar/bedrock-service-core/blob/main/lib/http/configs.js#L165-L175
https://github.com/digitalbazaar/bedrock-service-core/blob/main/lib/storage/ConfigStorage.js#L112-L148